osbs/conf.py

```diff
@@ -266,12 +266,23 @@ def get_registry_secrets(self):
         else:
             return []
 
-    def get_registry_api_versions(self):
+    def get_registry_api_versions(self, platform=None):
         value = self._get_value("registry_api_versions",
                                 self.conf_section,
                                 "registry_api_versions",
                                 default='v1,v2')
-        return value.split(',')
+        if not platform:
+            return value.split(',')
+        else:
```

Comment:
We could lose the `else`: here and de-indent the contained block. If `platform` is falsy, execution will not get here.